mongodbatlas/fw_resource_mongodbatlas_encryption_at_rest.go

```diff
@@ -225,28 +227,23 @@ func (r *EncryptionAtRestRS) Create(ctx context.Context, req resource.CreateRequ
 		encryptionAtRestReq.GoogleCloudKms = *newAtlasGcpKms(encryptionAtRestPlan.GoogleCloudKmsConfig)
 	}
 
-	var encryptionResp *matlas.EncryptionAtRest
+	stateConf := &retry.StateChangeConf{
+		Pending:    []string{retrystrategy.RetryStrategyPendingState},
+		Target:     []string{retrystrategy.RetryStrategyCompletedState, retrystrategy.RetryStrategyErrorState},
+		Refresh:    resourceMongoDBAtlasEncryptionAtRestCreateRefreshFunc(ctx, projectID, conn, encryptionAtRestReq),
+		Timeout:    1 * time.Minute,
+		MinTimeout: 1 * time.Second,
+		Delay:      0,
```

Comment:
I believe it makes sense to add a delay here to wait before retrying the operation.